org.srb2.SRB2Kart.json

```diff
@@ -0,0 +1,99 @@
+/* SPDX-License-Identifier: FSFAP
```

Comment:
I would avoid having a comment in the json file :) you can add a license to the repository if you want to

Reply:
ok.  Is there a validator tool?  `flatpak-builder` didn't complain...

But probably I will just switch to yaml, which I think explicitly allows comments.